package.json

```diff
@@ -222,6 +222,7 @@
         "flow-typed": "^2.6.2",
         "form-serialize": "^0.7.2",
         "formik": "^2.0.3",
+        "full-icu": "^1.3.1",
```

Comment:
Will this fix work in Japanese and other languages if the `full-icu` package is not installed? If not, then we need to add this to peer dependencies as well as the dev dependencies and add a message to the merge commit message like "WARNING: breaking change. This adds a new peer dependency on full-icu" so that BUIE customers will know. Also, you will need to submit a PR for adding `full-icu` to the dependencies of EUA and admin console so they don't break when they pick up the latest BUIE.

Reply:
The code will still work for Japanese and other languages if the full-icu package is not installed. The `full-icu` package is being added specifically for the test environment.

Reply:
Yep, it's just needed for the Intl global in Node. In the browser, the functionality is polyfilled or already available globally.

Reply:
ok cool